docs/includes/apiargs-MongoDBCollection-method-watch-option.yaml

```diff
@@ -1,6 +1,13 @@
-source:
-  file: apiargs-MongoDBCollection-method-aggregate-option.yaml
-  ref: batchSize
+---
+arg_name: option
+name: batchSize
+type: integer
+description: |
+  Specifies the maximum number of change events to return in each batch of the
+  response from the MongoDB cluster.
+interface: phpmethod
+operation: ~
+optional: true
```

Comment:
This documentation is correct for the spec itself, but PHP still doesn't support this (see: [PHPLIB-321](https://jira.mongodb.org/browse/PHPLIB-321)).